### PR TITLE
Edited ReadMe to add Docker instruction to Installation instructions

### PR DESCRIPTION
**Pull request by charisearter**
## Description ##

Clearly documents the reason for the pull request and what has changed. Delete this test and add your own.
- [ ] This is a bugfix.
- [ ] This is a new feature.
- [X] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?

## Review ##

- [x] Have you added two reviewers to this pull request?
- [ ] Is this a work in process?
- [x] Is this ready to be reviewed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ OKTA_URL_ISSUER=
 
 - run: `npm install` to download all dependencies.
 - run: `cp .env.sample .env` and update the enviornment variables to match your local setup.
+- run: `docker-compose up` * Will take a little while * (after installing Docker.)
```
Comment (VinceWilliamsDev) [inline README.md]: The instructions were a bit confusing without this line.

I would also note this line is meant to start the PostgreSQL DB.

Comment (charisearter) [reply README.md]: Okay. I'll do it now.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ OKTA_URL_ISSUER=
 
 - run: `npm install` to download all dependencies.
 - run: `cp .env.sample .env` and update the enviornment variables to match your local setup.
+- run: `docker-compose up` * Will start PostgresSQL DB and take a little while * (after installing Docker.)
```
Comment (VinceWilliamsDev) [inline README.md:43]: Nice update!

Comment (charisearter) [reply README.md:43]: Thanks!

**Review by Joe-Bay — APPROVED** [review]

**Pull request merged**